src/Stryker.Core/Stryker.Core/Mutants/ActiveMutantHelper.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Stryker
+{
+    public static class Helper
+    {
+        private static readonly int activeMutant;
```

Comment:
Shouldn't this be `int?`, since the int default value is 0. When no EnvironmentVariable is set, Mutant 0 would always be active. Or we could start the mutant ids at 1?

Reply:
Good point. I assumed that we always wanted to have an active mutant. I just added a commit that default to -1 instead of 0.